src/Microsoft.SqlTools.ServiceLayer/LanguageExtensibility/ExternalLanguageOperations.cs

```diff
@@ -275,32 +275,29 @@ private string GetLanguageModifyScript(ExternalLanguage language, List<ExternalL
                 }
             }
             return $@"
-{scriptAction} {language.Name}
+{scriptAction} [{language.Name}]
 {ownerScript}
 {contentAction} {contentScript}
 ";
         }
 
-        private string AddStringParameter(string paramName, string prefix, string postfix, string paramValue, Dictionary<string, object> parameters)
+        private string AddStringParameter(string paramName, string prefix, string postfix, string paramValue)
         {
             string script = string.Empty;
             
-            /*
             if (!string.IsNullOrWhiteSpace(paramValue))
             {
-                string sqlParam = $"{paramName}{postfix}";
-                script = $"{prefix} {paramName} = @{sqlParam}";
-                parameters.Add($"{sqlParam}", paramValue);
-            }
-            */
-            if (!string.IsNullOrWhiteSpace(paramValue))
-            {
-                script = $"{prefix} {paramName} = N'{paramValue}'";
+                script = $"{prefix} {paramName} = N'{EscapeStringForSql(paramValue)}'";
             }
             
             return script;
         }
 
+        private static string EscapeStringForSql(string value)
```

Comment:
We already have string escape functions : https://github.com/microsoft/sqltoolsservice/blob/master/src/Microsoft.SqlTools.ServiceLayer/Management/Common/Utils.cs#L542